README.md

```diff
@@ -86,33 +80,6 @@ This can be either a badge, a virtual item from the game's economy (sword, coins
 ## Getting Started
 > NOTE: LevelUp depends on SOOMLA's other modules: Core, Store, and Profile. This document assumes that you are new to SOOMLA and have not worked with any of the other SOOMLA modules. If this is not the case, and you already *have* some or all of the other modules, please follow these directions only for the modules you are missing and of course, for the **LevelUp** module.
 
-1. Download the following unitypackages:
```

Comment:
@Blinkq please make sure that there are complete instructions for integrating from GitHub.